native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/email-native/resource-config.json

```diff
@@ -0,0 +1,24 @@
+{
+  "resources":{
+  "includes":[
+    {
+      "pattern":"\\QMETA-INF/javamail.charset.map\\E"
+    }, 
+    {
+      "pattern":"\\QMETA-INF/javamail.default.address.map\\E"
+    }, 
+    {
+      "pattern":"\\QMETA-INF/mailcap.default\\E"
+    }, 
+    {
+      "pattern":"\\QMETA-INF/mailcap\\E"
+    }, 
+    {
+      "pattern":"\\QMETA-INF/services/javax.mail.Provider\\E"
+    },
+    {
+      "pattern":"\\Qorg/slf4j/impl/StaticLoggerBinder.class\\E"
```

Comment:
Can we try removing this logger class as well?

Reply:
Fixed with [b583bda1456a5bdf0c9fbe6336b93c208df89569](https://github.com/ballerina-platform/module-ballerina-email/pull/561/commits/b583bda1456a5bdf0c9fbe6336b93c208df89569)